src/OrleansSQLUtils/Storage/Provider/AdoGrainKey.cs

```diff
@@ -73,17 +74,18 @@ public byte[] GetHashBytes()
                 bytes = ToGuidKey(N0Key, N1Key).ToByteArray();
             }
 
+            var utf8 = new UTF8Encoding();
```

Comment:
@sergeybykov Ok it was not the ThreadLocal what we were moving away from, I was remembered incorrectly.